<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifyTeam</class>
 <widget class="QWidget" name="modifyTeam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #1b1f2a, stop:1 #10131a);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>541</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(25, 31, 42);
    color: #FFFFFF;
    font-family: 'Segoe UI', Arial, sans-serif;
    font-size: 12px;
    border-radius: 15px;
    padding: 15px;
    spacing: 2px;
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="10" column="0">
     <widget class="QPushButton" name="button1_2">
      <property name="styleSheet">
       <string notr="true">background-color: #00FF7F; /* Green to match the accent */
    color: #1A2633; /* Dark blue text for contrast */
    font-size: 12px;
    font-weight: bold;
    border-radius: 5px;
    padding: 6px 12px;
    margin-left: 350px;
    margin-right: 20px;
    margin-bottom: 5px;</string>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="cb_2">
      <property name="styleSheet">
       <string notr="true">background-color: #2A3240;
  border: 1px solid #3A4A5A;
    border-radius: 5px;
    padding: 5px;
    color: #E0E6ED;
    margin-left: 15px;
    margin-right: 15px;

Qcb::drop-down {
    border-left: 1px solid #3A4A5A;
    width: 20px;
}
Qcb:focus {
    border: 1px solid #00FF7F;
    background-color: #323A48;
}</string>
      </property>
      <item>
       <property name="text">
        <string extracomment="eu">bernabeu</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string extracomment="eu">Santiago Bernab√©u</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string extracomment="eu">Camp Nou</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string extracomment="eu">Old Trafford</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string extracomment="eu">Wembley Stadium</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_12">
      <property name="styleSheet">
       <string notr="true">color: #A0A8B7;
font-weight: bold;
padding-bottom: 5px;background-color:transparent;</string>
      </property>
      <property name="text">
       <string>         Team_Logo :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="l1_2">
      <property name="styleSheet">
       <string notr="true">background-color: #2A3240; /* Darker blue-gray for input fields */
    border: 1px solid #3A4A5A; /* Subtle blue-gray border */
    border-radius: 5px;
    padding: 5px;
    color: #E0E6ED; /* Light gray text for readability */
    margin-left: 15px;
    margin-right: 15px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_13">
      <property name="styleSheet">
       <string notr="true">color: #A0A8B7;
font-weight: bold;
padding-bottom: 5px;background-color:transparent;</string>
      </property>
      <property name="text">
       <string>         Budget :</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_14">
      <property name="styleSheet">
       <string notr="true">color: #A0A8B7;
font-weight: bold;
padding-bottom: 5px;background-color:transparent;</string>
      </property>
      <property name="text">
       <string>        Team_ABV :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_15">
      <property name="styleSheet">
       <string notr="true">color: #A0A8B7;
font-weight: bold;
padding-bottom: 5px;background-color:transparent;</string>
      </property>
      <property name="text">
       <string>          Team_Name :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_16">
      <property name="styleSheet">
       <string notr="true">color: #A0A8B7;
font-weight: bold;
padding-bottom: 5px;background-color:transparent;</string>
      </property>
      <property name="text">
       <string>         Home_Stadium :</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="l2_3">
      <property name="styleSheet">
       <string notr="true">background-color: #2A3240; /* Darker blue-gray for input fields */
    border: 1px solid #3A4A5A; /* Subtle blue-gray border */
    border-radius: 5px;
    padding: 5px;
    color: #E0E6ED; /* Light gray text for readability */
    margin-left: 15px;
    margin-right: 15px;</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QWidget" name="widget_3" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: #2A3240;
    border: 1px solid #3A4A5A;
    border-radius: 5px;
    padding: 3px;
    color: #E0E6ED;
    margin-left: 15px;
    margin-right: 15px;
    margin-bottom: 15px;</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QToolButton" name="toolButton_2">
         <property name="styleSheet">
          <string notr="true">background-color: #1C1F2A;
border: 1px solid #2A2D3A;
border-radius: 5px;
padding: 3px;
color: white;
margin-left: 15px;
margin-right:15px;
margin-bottom:15px;</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">color: #A0A8B7;
padding-left: 10px;
background-color: transparent;</string>
         </property>
         <property name="text">
          <string>insert logo here</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLineEdit" name="l2_4">
      <property name="styleSheet">
       <string notr="true">background-color: #2A3240; /* Darker blue-gray for input fields */
    border: 1px solid #3A4A5A; /* Subtle blue-gray border */
    border-radius: 5px;
    padding: 5px;
    color: #E0E6ED; /* Light gray text for readability */
    margin-left: 15px;
    margin-right: 15px;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>17</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../images.qrc">:/teamimg/terrain.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>346</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../images.qrc">:/teamimg/logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>18</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../images.qrc">:/teamimg/abv.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>12</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../images.qrc">:/teamimg/money.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>17</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #B0C0D0;background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../images.qrc">:/teamimg/Userr.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Style for the Adjust Team title label */
QLabel#label_4 {
    background-color: transparent; /* Dark background matching the app */
    color: #FFFFFF; /* White text for contrast */
    font-size: 25px; /* Larger font for title emphasis */
    font-weight: bold; /* Bold for a strong title appearance */


    padding: 5px 10px; /* Padding for a balanced look */
}</string>
   </property>
   <property name="text">
    <string> Modify Team</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
